src/core_plugins/metrics/public/visualizations/components/timeseries.js

```diff
@@ -68,7 +68,7 @@ class Timeseries extends Component {
               if (row.data[i] && pos.x < row.data[i][0]) break;
             }
             if (!row.data[closest]) return values[row.id] = null;
-            const [ time, value ] = row.data[closest];
+            const [ , value ] = row.data[closest];
```

Comment:
Maybe it should just be `const value = row.data[closest][1]`? I just found this a bit "confusing" (as in: I wasn't sure whether or not it was _actually_ legal, which I find to be a bad sign)

(However: I'm absolutely okey with keeping it, I think enabling this rule in eslint is much more important than discussing this case)